service/controller/v11/resource/deployment/master_deployment.go

```diff
@@ -19,6 +19,7 @@ func newMasterDeployments(customObject v1alpha1.KVMConfig) ([]*extensionsv1.Depl
 
 	privileged := true
 	replicas := int32(1)
+	podDeletionGracePeriod := int64(key.PodDeletionGracePeriod.Seconds())
```

Comment:
We prepare the master pods for draining support. This means the grace deletion period is extended to give workloads more time in the guest clusters. Below is described what else happens here. Note that all this applies to the workers as well.